integration-tests/cypress/e2e/matching.feature

```diff
@@ -1,3 +1,4 @@
+#***
```

Comment:
is this #*** meant to be here?

Reply:
Sorted, let's pair for the below comment